test/e2e/llmisvc/fixtures.py

```diff
@@ -71,32 +71,281 @@
                         "resources": {
                             "limits": {"cpu": "2", "memory": "10Gi"},
                             "requests": {"cpu": "1", "memory": "8Gi"},
-                        }
+                        },
                     }
                 ]
             }
-        }
+        },
     },
     "model-fb-opt-125m": {
         "model": {"uri": "hf://facebook/opt-125m", "name": "facebook/opt-125m"},
     },
+    "model-deepseek-v2-lite": {
+        "model": {
+            "uri": "hf://deepseek-ai/DeepSeek-V2-Lite-Chat",
+            "name": "deepseek-ai/DeepSeek-V2-Lite-Chat",
+        },
+    },
+    "workload-dp-ep-gpu": {
+        "replicas": 2,
+        "parallelism": {
+            "data": 1,
+            "dataLocal": 8,
+            "expert": True,
+            "tensor": 1,
+        },
+        "template": {
+            "containers": [
+                {
+                    "name": "main",
+                    "env": [
+                        {"name": "VLLM_LOGGING_LEVEL", "value": "DEBUG"},
+                        {"name": "TRITON_LIBCUDA_PATH", "value": "/usr/lib64"},
+                        {"name": "HF_HUB_DISABLE_XET", "value": "1"},
+                        {"name": "VLLM_SKIP_P2P_CHECK", "value": "1"},
+                        {"name": "VLLM_RANDOMIZE_DP_DUMMY_INPUTS", "value": "1"},
+                        {"name": "VLLM_USE_DEEP_GEMM", "value": "0"},
+                        {
+                            "name": "VLLM_ALL2ALL_BACKEND",
+                            "value": "deepep_high_throughput",
```

Comment:
why not keep the same pattern than the other envs?

Reply:
Good point